src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -457,10 +382,10 @@ protected function prepare_links( $id ) {
 	 *
```

Comment:
```suggestion
	 * @since 6.5.0 Add $post and $request parameters. 
	 *
	 * @param WP_Post         $post    Post object.
	 * @param WP_REST_Request $request Request object.
```

Reply:
@mukeshpanchal27 ^